books/RayTracingInOneWeekend.html

```diff
@@ -1064,19 +1064,20 @@
 
     // Common Headers
 
-    #include "common/ray.h"
-    #include "common/vec3.h"
+    #include "ray.h"
+    #include "vec3.h"
 
     #endif
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    [Listing [rtweekend-initial]: <kbd>[common/rtweekend.h]</kbd> The rtweekend.h common header]
+    [Listing [rtweekend-initial]: <kbd>[rtweekend.h]</kbd> The rtweekend.h common header]
```

Comment:
rtweekend seems like a solid candidate for leaving in common

Reply:
No headers move in this change. It simply eliminates the need to prefix headers with the 'common/' path. It also allows programmers to put the headers all in one source directory, or to use the source code as-is with some headers in common. Either way, the compiler will locate the header.